iis-wf/iis-wf-ingest-pmc/src/test/java/eu/dnetlib/iis/wf/ingest/pmc/metadata/MetadataImporterMain.java

```diff
@@ -29,30 +32,52 @@ public class MetadataImporterMain {
     
     private static final String encoding = "utf-8"; 
     
+    private static final String parsedFileSuffix = ".parsed";
```

Comment:
Wouldn't it be convenient for the output files to have ".parsed.json" extension? Eg. for automatic syntax highlighting in editors?